shared/chat/conversation/bottom-banner/index.tsx

```diff
@@ -109,11 +109,19 @@ const InviteBanner = ({users, openSMS, openShareSheet}: InviteProps) => {
   }
 
   const hasPhoneNumber = users.some(user => user.endsWith('@phone'))
+  const hasEmailAddress = users.some(user => user.endsWith('@email'))
   return (
     <BannerBox color={Styles.globalColors.blue}>
       <BannerText>
         Your messages will unlock once they join Keybase
-        {hasPhoneNumber && ' and verify their phone number'}.
+        {hasPhoneNumber
```

Comment:
let's derive this string earlier in the function

Reply:
k rdy